python/.gitignore

```diff
@@ -6,6 +6,7 @@ Makefile
 cmake_install.cmake
 build/
 Testing/
+.hypothesis/
```

Comment:
Can you put it under "Python stuff" below?

Reply:
Done.  I put it just under the coverage rule.